.cargo/config

```diff
@@ -1,5 +1,2 @@
-[build]
```

Comment:
This is needed so that the fluvio-websocket-proxy crate will work. I'm not a fan but don't have time to do it better.